### PR TITLE
client: add option for read/write stripe file as normal

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/client/io_tracker.h [hunk 1]
```diff
@@ -61,7 +61,8 @@ class CURVE_CACHELINE_ALIGNMENT IOTracker {
     IOTracker(IOManager* iomanager,
               MetaCache* mc,
               RequestScheduler* scheduler,
-              FileMetric* clientMetric = nullptr);
+              FileMetric* clientMetric = nullptr,
+              bool disableStripe = false);
```
Comment (ilixiaocui) [inline src/client/io_tracker.h:65]: Note to explain disableStripe

File: src/client/libcurve_file.h [hunk 2]
```diff
@@ -97,7 +97,8 @@ class FileClient {
      * @return: 返回文件fd
      */
     virtual int Open4ReadOnly(const std::string& filename,
-                              const UserInfo_t& userinfo);
+                              const UserInfo_t& userinfo,
+                              bool disableStripe = false);
```
Comment (ilixiaocui) [inline src/client/libcurve_file.h:101]: python cal need modify？

**Review by ilixiaocui — APPROVED** [review]

File: src/client/io_tracker.h (hunk 1 above)
Comment (wu-hanqing) [reply src/client/io_tracker.h:65]: explained in `IOManager4File`

File: src/client/libcurve_file.h (hunk 2 above)
Comment (wu-hanqing) [reply src/client/libcurve_file.h:102]: unnecessary, currently `Open4ReadOnly` only for chunkserver/client read clone source file.

Comment (wu-hanqing) [reply src/client/libcurve_file.h:102]: unnecessary, currently Open4ReadOnly only for chunkserver/client read clone source file.

File: src/client/io_tracker.h (hunk 1 above)
Comment (wu-hanqing) [reply src/client/io_tracker.h:65]: explained in IOManager4File

**Review by xu-chaojie — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**